src/order/cover.lean

```diff
@@ -34,12 +35,23 @@ lemma covers.lt (h : a ⋖ b) : a < b := h.1
 lemma not_covers_iff (h : a < b) : ¬a ⋖ b ↔ ∃ c, a < c ∧ c < b :=
 by { simp_rw [covers, not_and, not_forall, exists_prop, not_not], exact imp_iff_right h }
 
+/-- If `x < y` but `y` does not cover `x`, then there's an element in between. -/
+lemma exists_lt_lt_of_not_covers (hab : a < b) (h : ¬a ⋖ b) : ∃ c, a < c ∧ c < b :=
```

Comment:
This would be less confusing if you use the same names in the statement and comment.

Reply:
This is already in mathlib 😛